convertkit-api-ruby.gemspec

```diff
@@ -0,0 +1,19 @@
+require_relative 'lib/convertkit/version'
+
+Gem::Specification.new do |gem|
+  gem.name        = 'convertkit-api-ruby'
+  gem.version     = ConvertKit::VERSION
+  gem.platform    = Gem::Platform::RUBY
+  gem.authors     = ["Joel Megerssa"]
```

Comment:
No to take the credit from youJoel but should this be Mighty Networks ? 

cc: @timherby please let me know if I'm wrong

Reply:
Open to changing this to "Mighty Networks" if that's better.

Reply:
I think the gem publisher should be Mighty Networks (though obviously contributions to the repo will be clear by person who contributes to it.

Reply:
Done! https://github.com/untitledstartup/convertkit-api-ruby/pull/1/commits/da0c531ecd8a7bd98959593e3024490bdfa8c32e